### ISSUE TITLE
[Update] : [Legal] : EULA 

### ISSUE THREAD

**Issue by h0lybyte** [KBVE/kbve.com#191]
**Describe the update**
This issue is in reference to the `EULA` required by Steam and several other publishers.

I will create this issue as a reference point for the `EULA`. The best step I can think of right now? would be to use a legal boilerplate and then go through the proper channels once it is ready.

* * *

**References for update**
Include any links / data for the update that must be done.

[EULA](https://kbve.com/legal/#eula)

* * *

**Security/Performance risks**
Are there any major security and/or performance risks?!
The only risk is within the legal realm, which falls under security.

* * *

**Comment by h0lybyte** [comment]
I will close this out, the EULA can be found here :
https://kbve.com/legal/eula/

There needs to be some design work done on legal documents but that can be postpone until the MDX files can render. 

So future to:do will be to migrate all <md> files to <mdx>
